redirects/saas-15.5.txt

```diff
@@ -0,0 +1,17 @@
+# applications/finance
+
+applications/finance/payment_acquirers.rst applications/finance/payment_providers.rst
+applications/finance/payment_acquirers/adyen.rst applications/finance/payment_providers/adyen.rst
+applications/finance/payment_acquirers/alipay.rst applications/finance/payment_providers/alipay.rst
+applications/finance/payment_acquirers/authorize.rst applications/finance/payment_providers/authorize.rst
+applications/finance/payment_acquirers/buckaroo.rst applications/finance/payment_providers/buckaroo.rst
+applications/finance/payment_acquirers/flutterwave.rst applications/finance/payment_providers/flutterwave.rst
+applications/finance/payment_acquirers/mollie.rst applications/finance/payment_providers/mollie.rst
+applications/finance/payment_acquirers/ogone.rst applications/finance/payment_providers/ogone.rst
+applications/finance/payment_acquirers/paypal.rst applications/finance/payment_providers/paypal.rst
+applications/finance/payment_acquirers/sips.rst applications/finance/payment_providers/sips.rst
+applications/finance/payment_acquirers/stripe.rst applications/finance/payment_providers/stripe.rst
+applications/finance/payment_acquirers/wire_transfer.rst applications/finance/payment_providers/wire_transfer.rst
```

Comment:
- add the missing redirections
- add a note to explain the change (not compulsory but I'm used to do it ^^)
```suggestion
applications/finance/payment_acquirers.rst applications/finance/payment_providers.rst                                                 # /payment_acquirers -> /payment_providers
applications/finance/payment_acquirers/adyen.rst applications/finance/payment_providers/adyen.rst                                     # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/alipay.rst applications/finance/payment_providers/alipay.rst                                   # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_providers/amazon_payment_services.rst applications/finance/payment_providers/amazon_payment_services.rst # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_providers/asiapay.rst applications/finance/payment_providers/asiapay.rst                                 # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/authorize.rst applications/finance/payment_providers/authorize.rst                             # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/buckaroo.rst applications/finance/payment_providers/buckaroo.rst                               # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/flutterwave.rst applications/finance/payment_providers/flutterwave.rst                         # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/mercado_pago.rst applications/finance/payment_providers/mercado_pago.rst                       # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/mollie.rst applications/finance/payment_providers/mollie.rst                                   # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/ogone.rst applications/finance/payment_providers/ogone.rst                                     # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/paypal.rst applications/finance/payment_providers/paypal.rst                                   # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/sips.rst applications/finance/payment_providers/sips.rst                                       # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/stripe.rst applications/finance/payment_providers/stripe.rst                                   # /payment_acquirers/* -> /payment_providers/*
applications/finance/payment_acquirers/wire_transfer.rst applications/finance/payment_providers/wire_transfer.rst                     # /payment_acquirers/* -> /payment_providers/*

```